### PR TITLE
Add new macros to readme

### PR DESCRIPTION
**Pull request by joellabes**
## Description & motivation
rearrange the readme and add new macros

## Checklist
- [ ] I have verified that these changes work locally
- [x] I have updated the README.md (if applicable)
- [ ] I have added tests & descriptions to my models (and macros if applicable)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,46 +81,94 @@ Setting the `summarize` argument to `false` lets you check which rows do not mat
...
+    columns = None
 ) }}
 
 ```
 
-### compare_relations ([source](macros/compare_relations.sql))
+### quick_are_queries_identical ([source](macros/quick_are_queries_identical.sql))
 
-Similar around to `compare_relations`, except it takes two relations (instead of two queries).
+On supported adapters (currently Snowflake and BigQuery), take a hash of all rows in two queries and compare them.
```
Comment (graciegoheen) [inline README.md:131]: Would be nice to have a table up at the top that breaks down which macros are supported for which adapters

File: README.md [hunk 2]
```diff
@@ -81,46 +81,94 @@ Setting the `summarize` argument to `false` lets you check which rows do not mat
...
 
-Each relation must have the same columns with the same names, but they do not have to be in the same order. Use `exclude_columns` if some columns only exist in one relation.
+This can be calculated relatively quickly compared to other macros in this package and can efficiently provide reassurance that a refactor introduced no changes.
+
+#### Arguments
```
Comment (graciegoheen) [inline README.md:135]: In other macros docs we put arguments after output - total nit pick, but would be nice to be consistent here

**Review by graciegoheen — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (joellabes) [reply README.md:131]: This is the only one that's not universally supported, so it'd be a very boring table

File: README.md (hunk 2 above)
Comment (joellabes) [reply README.md]: only because it's you who's asking 😂

**Pull request merged**